examples/.bazelversion

```diff
@@ -0,0 +1 @@
+3.4.1
```

Comment:
Is this needed if it's at the root now?

Reply:
It's a "nice to have". It means you can take the examples directory and more or less treat it as it's own workspace. I only added `.bazelversion` to the root of the repo because that's where I'd expect it to be. It just gets copied there from the file in the repo root. I can delete it if you feel it shouldn't exist. (I'm torn so I'll go with whatever makes everyone happy 😄 )

Reply:
I don't have a strong feeling either way, but my worry is that it's a bit confusing. Nothing outside of examples uses `.bazelversion`, and it feels like having two implies that the repo-wide version might differ from the examples version.

Reply:
Ok, I removed it from the examples but left the top level `.bazelversion` file remains.